docs/sources/setup/install/helm/reference.md

```diff
@@ -2162,6 +2162,10 @@ null
 			<td>Storage config. Providing this will automatically populate all necessary storage configs in the templated config.</td>
 			<td><pre lang="json">
 {
+  "alibabacloud": {
```

Comment:
Please note the comment at the top of this file.

<!-- Autogenerated. Modify ../production/helm/loki/reference.md.gotmpl -->

Reply:
My bad, let me revert that.

Reply:
The changes still need to be documented, but they need to be documented in  `/production/helm/loki/reference.md.gotmpl` not in `docs/sources/setup/install/helm/reference.md`.  Then you have to run 
`make -C docs sources/installation/helm/reference.md` (which will update `docs/sources/setup/install/helm/reference.md ) and commit the generated changes.

Reply:
I'm not sure to understand what I am suppose to change in [`/production/helm/loki/reference.md.gotmpl`](https://github.com/grafana/loki/blob/main/production/helm/loki/reference.md.gotmpl). The file seems to only have a loop that generates the doc (reference.md) from the values themselves. I ran the make command [already](https://github.com/grafana/loki/pull/10553/commits/557b5bbb018141fd45bee73245d99bf2523ca2a9). I'm not sure why the result does not seem to be the expected result though.